addons/mail/models/mail_activity.py

```diff
@@ -504,6 +504,25 @@ def action_feedback_schedule_next(self, feedback=False):
             'target': 'new',
         }
 
+    def _prepare_next_activity_values(self):
```

Comment:
Could be great to add some tests in ``test_mail`` addons. We notably have some custom test models, you could add an override on ``mail.test.activity`` model and check it is effectively called and taken into consideration.